### ISSUE TITLE
CA1820 Does not trigger for string value > or < string.empty tests in VB.

### ISSUE THREAD

**Issue by adraut** [dotnet/roslyn-analyzers#1550]
#### Analyzer package
Microsoft.CodeQuality.Analyzers, version [2.6.0](https://www.nuget.org/packages/Microsoft.CodeQuality.Analyzers/2.6.0)

#### Analyzer
TestForEmptyStringsUsingStringLengthAnalyzer

#### Repro steps
```
Public Class StringTests
    Public Sub GreaterThanEmpty()
        Dim s As String = Nothing
        Dim isNotEmpty As Boolean = (s > "")
    End Sub

    Public Sub LessThanEmpty()
        Dim s As String = Nothing
        Dim isNotEmpty As Boolean = (s < "")
    End Sub
End Class
```

#### Expected behavior
CA1820 diagnostic is reported.

#### Actual behavior
CA1820 diagnostic is not reported.

**Comment by mavasani** [comment]
Current behavior matches legacy FXCop and we have retained it for back compatibility.
